### PR TITLE
github actions: Update pylint for Fedora 33

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes: #147

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -17,15 +17,15 @@ jobs:
...
             dependencies: pylint==2.6.0 PyGithub==1.53
             task: lint
-          - python-version: 3.8.6
+          - python-version: 3.9.2
             dependencies: black==20.8b1 isort==4.3.21
             task: fmt-travis
 
           # MANDATORY CHECKS USING LOWEST SUPPORTED INTERPRETER
-          - python-version: 3.6.8
+          - python-version: 3.9.2
```
Comment (mulkieran) [inline .github/workflows/main.yml:28]: We need to leave lowest supported where it is, we can't bump it.

**Review by mulkieran — COMMENTED** [review]
One request, otherwise looks good.

**Comment by mulkieran** [comment]
Looks good. We may want to put the code changes in immediately, as I don't think they conflict w/ current defaults.

**Comment by bgurney-rh** [comment]
OK.  I can create a "v3" PR to squash the Python version changes into one commit.

**Pull request closed without merging**